ir/v1.def

```diff
@@ -475,11 +475,13 @@ class AttributeRef : Expression {
 class CompilerOptions;
 #end
 
-class V1Program {
+class V1Program : ISimpleNamespace {
```

Comment:
Most P4-16 visitor passes won't work because the V1 IR has lots of classes which are not handled by the P4-16 visitors.